modules/persistent-storage-local-metrics.adoc

```diff
@@ -19,8 +19,13 @@
 
 * `lso_lv_provisioned_PV_count`: total number of provisioned PVs for `LocalVolume`
 
-To use these metrics, be sure to:
+To use these metrics, enable them by doing one of the following:
 
-* Enable support for monitoring when installing the Local Storage Operator.
+* When installing the Local Storage Operator from _OperatorHub_ in the web console, select the _Enable Operator recommended cluster monitoring on this Namespace_ checkbox.
 
```

Comment:
Maybe add this pic is better?
<img width="556" alt="Screenshot 2025-01-17 at 11 11 01" src="https://github.com/user-attachments/assets/281bef8f-8b2a-4cb2-abb4-333e4fd9d3a6" />

Reply:
Discussed in slack, we prefer to keep the text description, since screenshots tend to become outdated quickly.